frontend/src/pages/auth/login.tsx

```diff
@@ -245,7 +258,7 @@ export const LoginPage: FC = () => {
             </>
         }
         return <>
-            <InputWithlabel label={databaseType.id === DatabaseType.MongoDb ? "Host Name (or paste Connection URL)" : "Host Name"} value={hostName} setValue={handleHostNameChange} />
+            <InputWithlabel label={databaseType.id === DatabaseType.MongoDb || DatabaseType.Postgres ? "Host Name (or paste Connection URL)" : "Host Name"} value={hostName} setValue={handleHostNameChange} />
```

Comment:
Here you'll have to modify the check to be:

databaseType.id === DatabaseType.MongoDb || databaseType.id === DatabaseType.Postgres

otherwise it'll always evaluate to true.